modules/@apostrophecms/db/index.js

```diff
@@ -24,6 +24,13 @@
 // connection instance is created that reuses the same sockets,
 // and `uri`, `host`, `connect`, etc. are ignored.
 //
+// ### `versionCheck`
+//
+// If `true`, check to make sure the database does not belong to an
+// older, incompatible major release release of Apostrophe and exit if it does.
+// Defaults to `true`. You can set this to `false` to avoid an extra query at startup
```

Comment:
If the check is only at start up, what do we gain by ever turning this off?

Reply:
Apostrophe does a lot at startup. It's time-consuming, and enterprise clients have historically asked for options to tone it down where possible when you know what you're doing.

Reply:
👍